test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -86,10 +92,18 @@ describe('Switch Ethereum Chain for two dapps', function () {
           .withPreferencesControllerSmartTransactionsOptedOut()
           .build(),
         dappOptions: { numberOfDapps: 2 },
-        localNodeOptions: {
-          ...defaultGanacheOptions,
-          concurrent: [{ port: 8546, chainId: 1338 }],
-        },
+        localNode: ['anvil', 'anvil'],
+        localNodeOptions: [
+          {},
+          {
+            blockTime: 2,
+            vmErrorsOnRPCResponse: false,
+            mnemonic:
+              'phrase upgrade clock rough situate wedding elder clever doctor stamp excess tent',
+            port: 8546,
+            chainId: 1338,
+          },
+        ],
```

Comment:
this is the new implementation of the previous 'doubleGanache'/'concurrent' logic